modules/fscloud/variables.tf

```diff
@@ -65,3 +65,43 @@ variable "cbr_rules" {
   description = "The list of context-based restrictions rules to create."
   default     = []
 }
+
+########################################################################################################################
+# COS
+########################################################################################################################
+
+variable "cos_destination_name" {
+  type        = string
+  description = "The name of the IBM Cloud Object Storage destination which will be created for the storage of failed delivery events."
+  default     = "COS Destination"
+}
+
+variable "cos_bucket_name" {
+  type        = string
+  description = "The name of an existing Object Storage bucket to use for the storage of failed delivery events."
+  default     = null
+}
+
+variable "cos_instance_id" {
+  type        = string
+  description = "The ID of the Object Storage instance that contains the bucket that is specified in the `cos_bucket_name` variable. Required only if `cos_integration_enabled` is set to `true`."
+  default     = null
+}
+
+variable "skip_en_cos_auth_policy" {
+  type        = bool
+  description = "Whether an IAM authorization policy is created that permits all Event Notifications instances in the resource group to interact with your Object Storage instance. Set to `true` to use an existing policy. Ignored if `cos_integration_enabled` is set to `false`."
```

Comment:
This description is now out of date. The auth policy is scoped to the exact EN instance and bucket